src/components/oculus-touch-controls.js

```diff
@@ -150,6 +150,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       controller: 0,
       rotationOffset: data.rotationOffset !== -999 ? data.rotationOffset : isRightHand ? -90 : 90
     });
+    el.emit('controller-connected', {type: 'oculus-touch-controller'});
```

Comment:
the case @fernandojsg is same element with both oculus and vive components, need to know which injected; really the component reference, not the element